src/main/java/com/gradle/enterprise/bamboo/config/PersistentConfigurationManager.java

```diff
@@ -3,30 +3,45 @@
 import com.atlassian.bamboo.bandana.PlanAwareBandanaContext;
 import com.atlassian.bandana.BandanaManager;
 import com.atlassian.plugin.spring.scanner.annotation.imports.ComponentImport;
+import com.fasterxml.jackson.core.JsonProcessingException;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Component;
 
+import java.io.IOException;
 import java.util.Optional;
 
 @Component
 public class PersistentConfigurationManager {
 
-    private static final String KEY = "com.gradle.bamboo.plugins.ge.config";
+    static final String LEGACY_CONFIG_V0_KEY = "com.gradle.bamboo.plugins.ge.config";
+    static final String CONFIG_V1_KEY = "com.gradle.bamboo.plugins.ge.config.v1";
+    static final String CURRENT_CONFIG_KEY = CONFIG_V1_KEY;
```

Comment:
I don't see a value in having these 2 separate variables (V1 and current). At the moment there is no need in having a system that supports multiple versions. It's not clear if we would need it in the future. I would just keep it very simple. If we only use `CONFIG_V1_KEY` in this class I would only leave this constant here and remove the others.

Reply:
Keeping the `CURRENT_CONFIG` to avoid having to change the class body in the future